lib/_stream_readable.js

```diff
@@ -302,7 +302,8 @@ function chunkInvalid(state, chunk) {
 }
 
 
-// if it's past the high water mark, we can push in some more.
+// It simply checks the the state length with high water mark
```

Comment:
Typo: `the the`

Reply:
The previous sentence stated that we could push more if we were past the highWaterMark.
I would like to have a similar sentence there.
This function returns the result of `.push()`.